llama-index-core/tests/workflow/test_workflow.py

```diff
@@ -390,7 +390,7 @@ async def step(self, ev: StartEvent) -> StopEvent:
 
     workflow = DummyWorkflow()
     with pytest.raises(ValueError, match="The step raised an error!"):
-        await workflow.run_step()
+        await workflow.run()
```

Comment:
I think this test is specifically testing that errors get raised in stepwise mode (with this change, this test is the same as the one above it

Reply:
oopsies! made the change. thanks!